express-intro/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "package.json",
+  "version": "1.0.0",
+  "description": "my first package of json",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo 'test'"
+  },
+  "keywords": [
+    "[]"
+  ],
+  "author": "'',",
+  "license": "ISC"
+}
```

Comment:
Please install the `express` package to your `express-intro` app so it shows up in your `package.json` as a dependency.